### DISCUSSION TITLE
Is there a way to only display `zsh-autocomplete` on TAB?

### DISCUSSION THREAD

**Discussion by noomly** [marlonrichert/zsh-autocomplete#546] [Q&A] [answered]
It seems the answer to this discussion isn't working anymore: https://github.com/marlonrichert/zsh-autocomplete/discussions/383#discussioncomment-1800808. I personally find the way zsh-autocomplete displays its menu on any keystroke distracting, and it gets quite jarring when there is not enough space and the prompt jumps x lines up when writing `ls -`...
Is there a way to only trigger the menu on certain key presses? Like up and down arrow keys to bring up the history or TAB to bring up the completion menu? I haven't found anything in the default configuration file hence this new post.

Thanks for your work on zsh-autocomplete and zsh-snap!

**Comment by marlonrichert** [comment]


**Reply by rodhash** [reply]
After disabling async, do we need to define a new binding?

TAB doesn't seem to work after disabling async.

**Comment by marlonrichert** [comment, marked as answer]
This will disable normal async completion, but will leave other async completion (such as `^R`) intact:
```zsh
zstyle ':autocomplete:list-choices:*' ignored-input '*'
```

@noomly @rodhash
